content/applications/hr/time_off/my_time.rst

```diff
@@ -0,0 +1,107 @@
+=======
+My time
+=======
+
+The :guilabel:`My Time` section of the *Time Off* application houses all the various time off
+information for the signed-in user.
+
+This includes the main *Time Off* dashboard, which displays an overview of the user's various time
+off balances, as well as the user's time off requests and allocations.
```

Comment:
Technically, "time-off" should be hyphenated each time it is followed by a noun.
```suggestion
off balances, as well as the user's time-off requests and allocations.
```